### PR TITLE
add post with relation

### PR DESCRIPTION
**Pull request by h1alexbel**
@hizmailovich take a look, please

## PR-Codex overview
This PR adds functionality to create a new post and associate it with a user. 

### Detailed summary
- Added `Posts` interface to define methods for adding, retrieving and iterating through posts.
- Added `NeoPosts` class to implement `Posts` interface for Neo4j database.
- Added `TxPosts` class to provide transactional support for `Posts` operations.
- Added `RqPost` class to handle requests for creating new posts.
- Added `UserController` method to create a new post and associate it with a user.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by hizmailovich — APPROVED** [review]
lgtm

File: src/main/java/com/solvd/blog/request/RqPost.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import java.time.LocalDate;
+
+/**
+ * @author Aliaksei Bialiauski (abialiauski.dev@gmail.com)
+ * @since 0.0.0
+ */
+@RequiredArgsConstructor
+public final class RqPost implements Post {
+
+    private final Long id;
```
[the remark this answers is not in the document]
Comment (h1alexbel) [reply src/main/java/com/solvd/blog/request/RqPost.java:15]: good idea, we need to create a ticket then

[the remark this answers is not in the document]
Comment (hizmailovich) [reply src/main/java/com/solvd/blog/request/RqPost.java:15]: Created, assigned to me

**Comment by hizmailovich** [comment]
@rultor merge

**Pull request merged**